README.md

```diff
@@ -1,23 +1,200 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+gem install rails
+
+### Setup
+
+Clone this repository to your desired folder:
+  microverse
+### Install
+
+Install this project with: Blu Stacks
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+Trevor
+
+- GitHub: [@githubhandle](https://github.com/valt19)
+- Twitter: [@twitterhandle](https://twitter.com/trevor-odeke)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/trevo-odeke)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
 
-# Built With
-**Tech stack**
-**client:**
-  HTML and CSS
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-# Prerequisities:
-**Setup**
-Clone this repository to your desired folder and run it using a browser.
-**Usage**
-To run this project you need a code editor and browser
+<!-- LICENSE -->
 
-**Author:**
-  Trevor:
-     github: https://github.com/valt19
-     LinkedIn: https://www.linkedin.com/in/Trevor-Odeke
+## 📝 License <a name="MIT"> MIT</a>
 
-**Contributing:**
-Contributions, issues, and feature requests are welcome!
\ No newline at end of file
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [Optional] Dear @valt19, I see you made changes to your readme. This is great but please remember everything in your readme should be about the project and the author (you). feel free to leave it empty if you don't have the appropriate information to put inside a heading. I can see you r readme can be customized more, and I hop you will do 🙏

Reply:
thanx a lot 🙏